awesomemotive/affiliatewp-affiliate-area-shortcodes
# Added data sanitization to shortcode args

Issue #54 

This is just to ensure that the shortcode arguments are escaped properly before going ahead.
The test is just to certify if the shortcode runs properly after these addition.

Please test all shortcodes from the docs: https://affiliatewp.com/docs/affiliate-area-shortcodes-installation-and-usage/

Steps to test:

1. Install **Affiliate Area Shortcodes** addon;
2. Pick a shortcode and test one by one using the docs as reference, use the shortcode parameters as we have in our examples, but also, try to use some JS injection like this [affiliate_referrals status="'); alert(1); ('"], it should not display any JS alerts;
4. Try to use <script>alert(1)</script> inside any parameter or using as a content, example: [affiliate_referrals status="paid"]<script>alert(1)</script>[/affiliate_referrals], it should not display JS alerts.


**Housekeeping**


**Proposed changes**
